src/capi.zig

```diff
@@ -47,20 +45,23 @@ pub const Configuration = extern struct { method: u8, error_bound: f32 };
 export fn compress(
     uncompressed_values_array: UncompressedValues,
     compressed_values_array: *CompressedValues,
-    configuration: Configuration,
+    method_idx: u8,
+    configuration: [*:0]const u8,
 ) i32 {
     const uncompressed_values = uncompressed_values_array.data[0..uncompressed_values_array.len];
 
+    const configuration_slice: []const u8 = mem.span(configuration);
+
     // Returning 1 is equivalent to returning Error.UnknownMethod.
-    if (configuration.method > tersets.getMaxMethodIndex()) return 1;
+    if (method_idx > tersets.getMaxMethodIndex()) return 1;
```

Comment:
Why not write `index` instead of `idx` to make the code more clear? Again, I see no upsides to using shorthands instead of full words when naming variables, types, functions, etc. For me, the small amount of extra time needed to write and/or autocomplete and/or generate proper variables seems irrelevant compared to the benefit of making the code easier to read and thus easier to maintain.